app/Filament/Admin/Resources/ServerResource/Pages/CreateServer.php

```diff
@@ -287,6 +288,13 @@ public function form(Form $form): Form
                                     'sm' => 4,
                                     'md' => 4,
                                 ]),
+                            TextInput::make('avatar_url')
+                                ->label('Server Avatar')
+                                ->columnSpanFull()
+                                ->live(debounce: 500)
+                                ->url()
+                                ->formatStateUsing(fn ($state) => $state ?? (new UiAvatarsProvider())->get($this->getRecord()))
+                                ->prefix(fn ($state) => new HtmlString("<img src='$state' width='24vh' height='24vh' class='fi-avatar object-cover object-center rounded-md h-8 w-8 fi-tenant-avatar shrink-0'/>")),
```

Comment:
Use [`<x-filament::avatar>`](https://filamentphp.com/docs/3.x/support/blade-components/avatar) instead of `<img>`.